packages/delivery-x-domain-request/rollup.config.npm.mjs

```diff
@@ -2,5 +2,5 @@ import createRollupConfig from "../../.rollup/index.mjs";
 
 export default createRollupConfig({
   input: "src/delivery.ts",
-  external: ['@bugsnag/core', '@bugsnag/core/lib/json-payload']
+  external: [/node_modules/]
```

Comment:
so it this just a shorter way to add all the specific dependencies for each package?